.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.14.1
```

Comment:
in lock file use 0.14.2 https://github.com/ozeranskii/httptap/blob/main/uv.lock#L1219 - I suggest using 0.14.2 here